ivadomed/utils.py

```diff
@@ -619,6 +638,123 @@ def threshold_predictions(predictions, thr=0.5):
     return thresholded_preds
 
 
+def segment_volume(model_fname, model_metadata_fname, image_fname, roi_fname=None):
```

Comment:
i suggest to keep the convention fname_XXX

Reply:
Done :-)